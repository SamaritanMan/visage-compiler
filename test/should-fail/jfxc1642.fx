/*
 * @test/compile-error
 */

class A {
    var a = f;
    function f(g1 : String,
        g2 : String,
        g3 : String,
        g4 : String,
        g5 : String,
        g6 : String,
        g7 : String,
        g8 : String,
        g9 : String
        ) : String {
        "asfd"
    }
}

