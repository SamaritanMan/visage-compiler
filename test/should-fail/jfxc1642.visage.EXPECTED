test/should-fail/jfxc1642.visage:6: sorry, not implemented: functions with more than 8 parameters
    var a = f;
            ^
1 error
